backend/routes/stories.js

```diff
@@ -23,20 +244,6 @@ router.get('/individualstory', async (req, res) => {
     }
 })
 
```

Comment:
The below endpoint was identified to be causing an issue where it would catch incoming requests meant for other endpoints due to the colon inside the URL. Further, we saw that this endpoint has duplicated functionality as `individualStory`, hence its ultimate removal here.